modules/host/host.go

```diff
@@ -199,6 +486,12 @@ func (h *Host) Close() error {
 	h.closed = true
 	h.resourceLock.Unlock()
 
+	// Close the bolt database.
+	err = h.db.Close()
+	if err != nil {
+		return err
+	}
```

Comment:
Should probably delay returning the error; don't want to stop short before clearing the port, saving the state, etc.